internal/metastore/kv/kv_catalog.go

```diff
@@ -441,31 +604,71 @@ func (kc *Catalog) ListCollections(ctx context.Context, ts typeutil.Timestamp) (
 			log.Warn("unmarshal collection info failed", zap.Error(err))
 			continue
 		}
-		colls[collMeta.Schema.Name] = model.UnmarshalCollectionModel(&collMeta)
+		collection, err := kc.GetCollectionByID(ctx, collMeta.GetID(), ts)
+		if err != nil {
+			return nil, err
+		}
+		colls[collMeta.Schema.Name] = collection
 	}
 
 	return colls, nil
 }
 
-func (kc *Catalog) ListAliases(ctx context.Context) ([]*model.Collection, error) {
-	_, values, err := kc.Snapshot.LoadWithPrefix(CollectionAliasMetaPrefix, 0)
+func (kc *Catalog) listAliasesBefore210(ctx context.Context, ts typeutil.Timestamp) ([]*model.Alias, error) {
+	_, values, err := kc.Snapshot.LoadWithPrefix(CollectionAliasMetaPrefix, ts)
 	if err != nil {
-		log.Error("get aliases meta fail", zap.String("prefix", CollectionAliasMetaPrefix), zap.Error(err))
 		return nil, err
 	}
+	// aliases before 210 stored by CollectionInfo.
+	aliases := make([]*model.Alias, 0, len(values))
+	for _, value := range values {
+		coll := &pb.CollectionInfo{}
+		err := proto.Unmarshal([]byte(value), coll)
+		if err != nil {
+			return nil, err
+		}
+		aliases = append(aliases, &model.Alias{
+			Name:         coll.GetSchema().GetName(),
+			CollectionID: coll.GetID(),
+			CreatedTime:  0, // not accurate.
```

Comment:
Better to change this to collection's createTime.
you can change this in next pr.